README.md

```diff
@@ -11,6 +10,27 @@ This tool will provide following features:
 * Mapping of the same landmarks between different images based on SLAM data association
 * Editing and fine-tuning landmarks positions and association
 
+## How do I create a project of the correct format?
+
+1. Create the ```Images``` folder, in which we put all the frames from the dataset. Images must match the following
+   criteria:
+    * 640x480 format
```

Comment:
concrete size is not required, images can be of any size, say that all images should have the same size.